### PR TITLE
PLANNER-2641 Fail fast on native DRL + cleanup

### PR DESCRIPTION
**Pull request by triceo**

### CODE REVIEW

File: optaplanner-quarkus-integration/optaplanner-quarkus/deployment/src/main/java/org/optaplanner/quarkus/deployment/OptaPlannerProcessor.java [hunk 1]
```diff
@@ -156,17 +157,19 @@ DetermineIfNativeBuildItem ifNativeBuild() {
...
             solverConfigIO.write(solverConfig, effectiveSolverConfigWriter);
             return new DevConsoleRuntimeTemplateInfoBuildItem("solverConfigProperties",
-                    new OptaPlannerDevUIPropertiesSupplier(effectiveSolverConfigWriter.toString()));
+                    new OptaPlannerDevUIPropertiesSupplier(effectiveSolverConfigWriter.toString()), this.getClass(),
+                    curateOutcomeBuildItem);
         } else {
             return new DevConsoleRuntimeTemplateInfoBuildItem("solverConfigProperties",
-                    new OptaPlannerDevUIPropertiesSupplier());
+                    new OptaPlannerDevUIPropertiesSupplier(), this.getClass(), curateOutcomeBuildItem);
         }
```
Comment (triceo) [inline optaplanner-quarkus-integration/optaplanner-quarkus/deployment/src/main/java/org/optaplanner/quarkus/deployment/OptaPlannerProcessor.java:173]: These constructors were deprecated, I replaced them with the recommended ones.

**Review by ge0ffrey — APPROVED** [review]
Looks good for me. Given that Chris is our gizmo expert, I'd love a review from him too.

**Review by Christopher-Chianelli — APPROVED** [review]
LGTM

**Comment by Christopher-Chianelli** [comment]
(There were no Gizmo related changed; only removed unused parameters to code generation methods)

**Pull request merged**